2021-02-24 10:28:51,224 INFO    MainThread:5696 [wandb_setup.py:_flush():70] setting env: {}
2021-02-24 10:28:51,224 INFO    MainThread:5696 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-02-24 10:28:51,224 INFO    MainThread:5696 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-24 10:28:51,224 INFO    MainThread:5696 [wandb_setup.py:_flush():70] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f6902a3eb80>
2021-02-24 10:28:51,224 INFO    MainThread:5696 [wandb_init.py:_log_setup():319] Logging user logs to /home/johannbs/Desktop/IceCube_GNN/wandb/run-20210224_102851-1g7t4zux/logs/debug.log
2021-02-24 10:28:51,224 INFO    MainThread:5696 [wandb_init.py:_log_setup():320] Logging internal logs to /home/johannbs/Desktop/IceCube_GNN/wandb/run-20210224_102851-1g7t4zux/logs/debug-internal.log
2021-02-24 10:28:51,224 INFO    MainThread:5696 [wandb_init.py:init():351] calling init triggers
2021-02-24 10:28:51,224 INFO    MainThread:5696 [wandb_init.py:init():356] wandb.init called with sweep_config: {}
config: {}
2021-02-24 10:28:51,225 INFO    MainThread:5696 [wandb_init.py:init():404] starting backend
2021-02-24 10:28:51,254 INFO    MainThread:5696 [backend.py:ensure_launched():81] starting backend process...
2021-02-24 10:28:51,280 INFO    MainThread:5696 [backend.py:ensure_launched():85] started backend process with pid: 5855
2021-02-24 10:28:51,281 INFO    MainThread:5696 [wandb_init.py:init():413] backend started and connected
2021-02-24 10:28:51,282 INFO    MainThread:5696 [wandb_init.py:init():436] updated telemetry
2021-02-24 10:28:51,283 INFO    MainThread:5696 [wandb_init.py:init():459] communicating current version
2021-02-24 10:28:52,029 INFO    MainThread:5696 [wandb_init.py:init():464] got version response upgrade_message: "wandb version 0.10.20 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-02-24 10:28:52,029 INFO    MainThread:5696 [wandb_init.py:init():472] communicating run to backend with 30 second timeout
2021-02-24 10:28:52,236 INFO    MainThread:5696 [wandb_init.py:init():495] starting run threads in backend
2021-02-24 10:28:53,811 INFO    MainThread:5696 [wandb_run.py:_console_start():1411] atexit reg
2021-02-24 10:28:53,811 INFO    MainThread:5696 [wandb_run.py:_redirect():1274] redirect: SettingsConsole.WRAP
2021-02-24 10:28:53,811 INFO    MainThread:5696 [wandb_run.py:_redirect():1309] Wrapping output streams.
2021-02-24 10:28:53,812 INFO    MainThread:5696 [wandb_run.py:_redirect():1325] Redirects installed.
2021-02-24 10:28:53,812 INFO    MainThread:5696 [wandb_init.py:init():518] run started, returning control to user process
2021-02-24 10:28:57,974 INFO    MainThread:5696 [wandb_config.py:__setitem__():142] config set hidden_states = 32 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f690036da60>>
2021-02-24 10:28:57,975 INFO    MainThread:5696 [wandb_run.py:_config_callback():663] config_cb hidden_states 32 None
2021-02-24 10:28:57,975 INFO    MainThread:5696 [wandb_config.py:__setitem__():142] config set forward = False - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f690036da60>>
2021-02-24 10:28:57,975 INFO    MainThread:5696 [wandb_run.py:_config_callback():663] config_cb forward False None
2021-02-24 10:28:57,975 INFO    MainThread:5696 [wandb_config.py:__setitem__():142] config set dropout = 0.25 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f690036da60>>
2021-02-24 10:28:57,975 INFO    MainThread:5696 [wandb_run.py:_config_callback():663] config_cb dropout 0.25 None
2021-02-24 10:28:57,975 INFO    MainThread:5696 [wandb_config.py:__setitem__():142] config set learning_rate = 0.0001 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f690036da60>>
2021-02-24 10:28:57,976 INFO    MainThread:5696 [wandb_run.py:_config_callback():663] config_cb learning_rate 0.0001 None
2021-02-24 10:28:57,976 INFO    MainThread:5696 [wandb_config.py:__setitem__():142] config set batch_size = 1024 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f690036da60>>
2021-02-24 10:28:57,976 INFO    MainThread:5696 [wandb_run.py:_config_callback():663] config_cb batch_size 1024 None
2021-02-24 10:28:57,976 INFO    MainThread:5696 [wandb_config.py:__setitem__():142] config set loss_func = likelihood_covariant_unitvectors - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f690036da60>>
2021-02-24 10:28:57,976 INFO    MainThread:5696 [wandb_run.py:_config_callback():663] config_cb loss_func likelihood_covariant_unitvectors None
2021-02-24 10:28:57,976 INFO    MainThread:5696 [wandb_config.py:__setitem__():142] config set n_neighbors = 6 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f690036da60>>
2021-02-24 10:28:57,977 INFO    MainThread:5696 [wandb_run.py:_config_callback():663] config_cb n_neighbors 6 None
2021-02-24 10:29:39,776 INFO    MainThread:5696 [wandb_init.py:_pause_backend():240] pausing backend
2021-02-24 10:29:52,379 INFO    MainThread:5696 [wandb_init.py:_resume_backend():245] resuming backend
2021-02-24 10:29:52,408 INFO    MainThread:5696 [wandb_init.py:_pause_backend():240] pausing backend
2021-02-24 10:31:16,251 INFO    MainThread:5696 [wandb_init.py:_resume_backend():245] resuming backend
2021-02-24 10:31:16,660 INFO    MainThread:5696 [wandb_init.py:_pause_backend():240] pausing backend
2021-02-24 10:31:18,970 INFO    MainThread:5696 [wandb_init.py:_resume_backend():245] resuming backend
2021-02-24 10:31:18,971 INFO    MainThread:5696 [wandb_init.py:_pause_backend():240] pausing backend
2021-02-24 10:31:18,973 INFO    MainThread:5696 [wandb_run.py:_atexit_cleanup():1381] got exitcode: 0
2021-02-24 10:31:18,973 INFO    MainThread:5696 [wandb_run.py:_restore():1353] restore
2021-02-24 10:31:19,209 INFO    MainThread:5696 [wandb_run.py:_wait_for_finish():1504] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 804
}

2021-02-24 10:31:21,213 INFO    MainThread:5696 [wandb_run.py:_wait_for_finish():1504] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 20557
  total_bytes: 20557
}

2021-02-24 10:31:23,876 INFO    MainThread:5696 [wandb_run.py:_show_files():1726] logging synced files
